cmd/root.go

```diff
@@ -63,6 +63,7 @@ You can also explicitly specify the scheme to use:
     {{.appName}} sbom:path/to/syft.json                 read Syft JSON from path on disk
     {{.appName}} registry:yourrepo/yourimage:tag        pull image directly from a registry (no container runtime required)
     {{.appName}} att:attestation.json --key cosign.pub  explicitly use the input as an attestation
+    {{.appName}} csv:path/to/yourcsv                    read a CSV of CPE,PURL from a path on disk
```

Comment:
Thanks for updating the documentation!

Reply:
@wagoodman we might have talked about this offline, but what should this be renamed too if we don't want to fully support `cve` parsing as a grype input?